src/apps/backend/modules/account/rest_api/account_view.py

```diff
@@ -24,11 +24,14 @@ def post(self) -> ResponseReturnValue:
             phone_number_data = request_data["phone_number"]
             phone_number_obj = PhoneNumber(**phone_number_data)
             account_params = CreateAccountByPhoneNumberParams(phone_number=phone_number_obj)
-            account = AccountService.get_or_create_account_by_phone_number(params=account_params)
+            account, otp = AccountService.get_or_create_account_by_phone_number(params=account_params)
```

Comment:
This is not OK.

`get_or_create_account_by_phone_number` as the name suggest should return only account. You are making code highly unreadable with this change